data/tutorials/gs_00_up_and_running.md

```diff
@@ -27,90 +27,119 @@ for the instruction.
+If you have installed the binary distribution of opam through the install script, this step should already be done. If you have installed it through your system package manager, you must initialise opam by running the following command:
 
-```
-opam init          # Can take some time
-eval $(opam env)
+```shell
+$ opam init          # Can take some time
+$ eval $(opam env)
 ```
 
-The first command (`opam init`) creates a first switch, usually called `default`, although this is just a convention. If you have installed OCaml through your system package manager, the first switch will be set up to use this compiler (it is called a "system switch"). Otherwise, it will build one from source, usually taking the most recent version of OCaml.
+The first command (`opam init`) creates a first switch, usually called `default`, although this is just a convention. (`switch` is used to have several installations on disk, like packages, compiler version, etc.) If you have installed OCaml through your system package manager, the first switch will be set up to use this compiler (it is called a "system switch"). Otherwise, it will build one from source, usually taking the most recent version of OCaml.
 
 The second command (`eval $(opam env)`) modifies a few environments variables to make the shell aware of the switch you are using. For instance, it will add what is needed to the `PATH` variable so that typing `ocaml` in the shell runs the OCaml binary of the current switch.
 
-In case you are not satisfied with the OCaml version of your system switch, you can write the following commands to create a new switch with a recent version of OCaml:
+**Please note:** After the `opam init` command, you might get a result asking if you'd like to update your `zsh` configuration. If you get that message, type in `N`, the default, then type `y` to install the hook, which will run `eval $(opam env)`. As you get more well-versed in OCaml, you can change these settings by rerunning `opam init`.
+
+Now check the installation by running `opam --version`. The current version is 2.1.2. Please note merely using `opam init` might install a previous version of opam. If you have a previous version, please refer to [the opam Upgrade Guide](https://opam.ocaml.org/doc/Upgrade_guide.html).
+
+### Install Compiler 
+
+Next, we'll install the OCaml base compiler, which uses the `opam switch create` command. Specify which version as shown below (i.e., `opam switch create 5.0.0~alpha1`). All possible compiler versions can be found with `opam switch list-available`. The most current version can be found at [opam.ocaml.org](https://opam.ocaml.org/packages/ocaml-base-compiler/).
 
 ```shell
-opam switch create 4.14.0
-eval $(opam env)
+$ opam switch create 5.0.0~alpha1
```

Comment:
When this doc makes it into the website, ocaml 5 will be released, no?

Reply:
I... guess? (but only time will tell!)

Reply:
😂  .... I changed them to 5.0.0 for now